src/CONST.ts

```diff
@@ -852,6 +852,9 @@ const CONST = {
         // The minimum number of typed lines needed to enable the full screen composer
         FULL_COMPOSER_MIN_LINES: 3,
     },
+    REPORT_NAME_KEYS: {
```

Comment:
Yes, but rNVPs should be _a bit simpler since we always want to send all of them, right?

Anyway, I assume it should be easy to do by queuing the update at the lowest level in auth, probably in [Report::setNameValuePair](https://github.com/Expensify/Auth/blob/3a48f959ca04971d7d98ff7872bd2f18a76a41ff/auth/lib/Report.cpp#L2296)